src/main/java/me/minidigger/hangar/controller/UsersController.java

```diff
@@ -134,9 +138,18 @@ public Object setLocked(@PathVariable Object user, @PathVariable Object locked,
         return null; // TODO implement setLocked request controller
     }
 
-    @RequestMapping("/{user}/settings/tagline")
-    public Object saveTagline(@PathVariable Object user) {
-        return null; // TODO implement saveTagline request controller
+    @RequestMapping(value = "/{user}/settings/tagline", method = RequestMethod.POST)
+    public ModelAndView saveTagline(@PathVariable String user, @RequestParam("tagline") String tagline) {
+        if (tagline.length() > hangarConfig.getMaxTaglineLen()) {
+            ModelAndView mav = showProjects(user);
+            AlertUtil.showAlert(mav, AlertUtil.ERROR, "error.tagline.tooLong"); // TODO pass length param to key
+            return showProjects(user);
+        }
+        // TODO user action log
+        UsersTable usersTable = userDao.get().getByName(user);
+        usersTable.setTagline(tagline);
+        usersTable = userDao.get().update(usersTable);
+        return showProjects(usersTable);
```

Comment:
I think rather than calling the method, we should do a forward here

Reply:
wouldn't that result in another query to the database? I was trying to avoid redundant queries by kinda pulling out most of the showProjects request mapping.